Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    atlas-api (0.1.2)
+    atlas-api (1.0.0)
```

Comment:
Possibly overthinking things, but moving from a 0.x to a 1.x conveys API stability that we may not want to guarantee. I'm not 100% sure if bumping the major version is necessary if we're dropping language version support, but not making changes to the API. We might want to ask around in our engineering channels for guidance.

Reply:
That's a good point. I was thinking of it in terms of the Tools team taking ownership of active development, but I agree that we haven't made API changes or guaranteed stability there. I'll drop back to 0.2.0.